sdk/eventhub/event-hubs/src/util/connectionStringUtils.ts

```diff
@@ -114,4 +121,14 @@ function validateProperties(
       "Connection string with SharedAccessKeyName should have SharedAccessKey as well.",
     );
   }
+
+  if (useDevelopmentEmulator === "true" && !isLoopbackAddress(endpoint)) {
+    throw new Error(
+      "Connection string should either has a loopback address or not have UseDevelopmentEmulator.",
+    );
+  }
+}
+
+function isLoopbackAddress(address: string) {
```

Comment:
Any way to use the one in Core rather than duplicating?

Reply:
Good point, addressed in https://github.com/Azure/azure-sdk-for-js/pull/29195/commits/55799d8b3c5648dd001128e293e8b2f94fc7b013.